.github/ISSUE_TEMPLATE/01-bug.yml

```diff
@@ -0,0 +1,60 @@
+name: General Bug
+labels: ["status/needs-triage", "type/bug"]
+description: Report a general problem with Bottlerocket
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This bug report form is for **general issues** with Bottlerocket. 
+        
+        A few tips before submitting a new issue:
+        - Search for similar issues, including [closed issues](https://github.com/bottlerocket-os/bottlerocket/issues?q=is%3Aissue+is%3Aclosed),
+        - Include details about the environment you're running in,
+        - Include any error messages you received, with any required context.
+  - type: dropdown
+    id: variant-family
+    attributes:
+      label: Variant family
+      options:
+        - aws-k8s
+        - aws-k8s (nvidia)
+        - aws-ecs
+        - aws-ecs (nvidia)
+        - vmware-k8s
+        - metal-k8s
+        - Custom variant
+  - type: input
+    id: variant-orchestrator-version
+    attributes:
+      label: Orchestrator version
+      description: For k8s variants, this is your version of Kubernetes. For ECS this is likely "1"
```

Comment:
It feels like the data we really need is:
* specific variant tuple
* specific Bottlerocket version

I'm not sure if having the variant family drop-down is helping very much. If they just put "1" here after selecting "aws-k8s" then it's overall worse for developers than if we just ask them to put the full variant name and give examples ("metal-k8s-1.24", "vmware-k8s-1.25", "aws-ecs-1").

Reply:
@bcressey Should we ask about architecture too?

Reply:
Architecture can be pretty significant - various daemonsets might not support aarch64 yet.